package.json

```diff
@@ -8,9 +8,11 @@
     "@deskpro/js-utils": "^1.0.4",
     "@deskpro/react-datepicker-hijri": "^1.4.2-beta.1",
     "@storybook/addon-knobs": "^3.4.3",
+    "ajv": "^6.5.2",
```

Comment:
That should be a dev dependency, it is not used outside the tests

Reply:
I had an idea to use this validation in the LayoutConfig helper in order to warn developers that the layout is not valid. But the library is a bit heavy so I decided to not implement that. What do you think?

Reply:
Probably better to keep things lightweight. I'm already worried by `moment` size ...

Reply:
we can replace `moment` with `date-fns`...

Reply:
I will look into that. Does it support hijri calendar?